.circleci/config.yml

```diff
@@ -205,24 +205,12 @@ jobs:
       SWIFT_VERSION: "5.4"
     <<: *ubuntu_tests_steps
 
-  "Run SwiftLint and Danger":
-    macos:
-      xcode: "11.4.0"
+  "Execute tests on Ubuntu 18.04 (Swift 5.5)":
```

Comment:
You forgot to add this job below to the `jobs` list.

Reply:
Great catch! These CircleCI configs are somewhat more confusing than GHA configs to me. I wonder if it's worth considering a migration from CircleCI to GHA for Linux jobs at least, at some point in the future.

Reply:
I agree, I was thinking about this too.